src/OnboardingSPA/steps/SiteGen/SiteDetails/index.js

```diff
@@ -78,7 +79,14 @@ const SiteGenSiteDetails = () => {
 			currentData.sitegen.homepages.data = {};
 			setCurrentOnboardingData( currentData );
 		}
-		setIsFooterNavAllowed( isValidInput );
+
+		if ( customerInput !== undefined && customerInput?.trim() !== '' ) {
```

Comment:
Is it safe to remove the `?` since we already checked for `undefined`?

Reply:
Sure.

Reply:
Added changes, Please check, Thanks.